Modelica/Mechanics/MultiBody/Sensors.mo

```diff
@@ -1541,7 +1541,9 @@ v_rel   = MultiBody.Frames.resolveRelative(frame_a.R, frame_b.R, v_rel_a);
   end RelativeVelocity;
 
   model RelativeAngles "Measure relative angles between two frame connectors"
-    extends Internal.PartialRelativeSensor;
+    extends Internal.PartialRelativeSensor
+      annotation (
+        IconMap(extent={{-100,-100}, {100,100}},primitivesVisible=false));
```

Comment:
This would be the first occurence of the IconMap annotation in a extends clause in the MSL. Though legal (in MLS 3.2r2), I am not sure if we should introduce it now, especially since it is not properly supported by all tools. For example, see [om:#4723](https://trac.openmodelica.org/OpenModelica/ticket/4723) for such a tool issue.

Reply:
I understand. The solution is legal but undesired at the moment.
I see two workarounds possible:
1. to revert RelativeAngles to v3.2.2 implementation,
2. to make that dotted line conditionally visible by a protected parameter.

Reversion is simple but the idea of inheritance is broken then.

Any preference?

Reply:
I would wait for @eshmoylova review, if IconMap works (beside Dymola and SimulationX) in MapleSim. If yes, fine for me. If no, revert to v3.2.2 implementation and apply all current changes for next major release.

Reply:
What about keeping IconMap - for primitivesVisible but without extent that seems redundant anyway? The worst case for a tool would be that a tool does not support primitivesVisible - but that just means the line is visible even if it shouldn't be.

Reply:
> without extent that seems redundant anyway

True.